src/views/homepage/components/featured-content-grid/search-featured-card.tsx

```diff
@@ -40,10 +49,13 @@ const SearchFeaturedCard = () => {
 	/**
 	 * If animation is enabled, set up the animation interval and event listeners
 	 * that are used to disable the animation with certain interactions.
+	 *
+	 * If the animation has been disabled, set up event listeners that are used to
+	 * re-enable the animation with certain interactions.
```

Comment:
🗒️ We received feedback that the permanent disabling of the animation was unexpected, so now it gets re-enabled after interacting with it. 😊